index.html

```diff
@@ -254,22 +254,18 @@
   <p>The application can collect, process, and act on the provided metrics via the provided JavaScript interface:</p>
 
   <pre class="example js">
-    // Gather server timing for a particular resource
-    const serverTimings = window.performance
-      .getEntriesByName("https://example.com/resource.jpg", "resource")
-      // Get just the serverTiming array
-      .map(({ serverTiming }) => serverTiming);
-
-    serverEntries
-      // we only care about "slow" ones
-      filter({ duration }  => duration > 2.0)
-      // process just the attributes we are interested in
-      .forEach(({ name, duration, description }) => {
-        // ...
-      });
-
-    // Aternatively, we convert each into JSON
-    const entriesAsJson = serverEntries.map(entry => entry.toJSON());
+    // serverTiming entries can live on 'navigation' and 'resource' entries
+    window.performance.getEntries()
```

Comment:
Drop window, since it may be used inside worker as well.

Also, experience shows that developers copy-paste examples, so we should spell out the best practices.. Grabbing *all* entries is probably not something we want to encourage? I think it's OK to query just one -- e.g. performance.getEntriesByType("resource"), with a note above it that serverTiming is also available on navigation entries.